app/scripts/lib/multichain-api/scope/authorization.ts

```diff
@@ -37,15 +31,67 @@ export const processScopes = (
   const flattenedRequiredScopes = flattenMergeScopes(validRequiredScopes);
   const flattenedOptionalScopes = flattenMergeScopes(validOptionalScopes);
 
-  assertScopesSupported(flattenedRequiredScopes, {
-    findNetworkClientIdByChainId,
-  });
-  assertScopesSupported(flattenedOptionalScopes, {
-    findNetworkClientIdByChainId,
-  });
-
   return {
     flattenedRequiredScopes,
     flattenedOptionalScopes,
   };
 };
+
+export const bucketScopes = (
```

Comment:
With a bunch of these helpers but I think especially this one it will be useful to have a nice description/explanation of what we're doing here.

Reply:
Happy to help iron out that language

Reply:
adding this to the slack todo